hudi-common/src/main/java/org/apache/hudi/metadata/FileSystemBackedTableMetadata.java

```diff
@@ -169,9 +170,9 @@ private List<String> getPartitionPathWithPathPrefixUsingFilterExpression(String
 
       // List all directories in parallel
       engineContext.setJobStatus(this.getClass().getSimpleName(), "Listing all partitions with prefix " + relativePathPrefix);
-      List<FileStatus> dirToFileListing = engineContext.flatMap(pathsToList, path -> {
```

Comment:
@CTTY looks like in the latest master, we no longer return `FileStatus` here (the `Path` instances are used instead).  Is this PR still needed?

Reply:
You are right, I don't think this is needed anymore

Reply:
Got it.  I'll close this PR.